test-tap/helper/report.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
-const childProcess = require('child_process');
+const cp = require('child_process');
+const wt = require('worker_threads');
```

Comment:
Would you mind sticking to the camelcased module names?

Reply:
Sure.